x-pack/solutions/security/plugins/security_solution/public/siem_migrations/rules/utils/translation_results/index.ts

```diff
@@ -12,11 +12,20 @@ import * as i18n from './translations';
 
 export const useResultVisColors = () => {
   const { euiTheme } = useEuiTheme();
+  if (euiTheme.themeName === 'EUI_THEME_AMSTERDAM') {
+    return {
+      [RuleTranslationResult.FULL]: euiTheme.colors.vis.euiColorVis0,
+      [RuleTranslationResult.PARTIAL]: euiTheme.colors.vis.euiColorVis5,
+      [RuleTranslationResult.UNTRANSLATABLE]: euiTheme.colors.vis.euiColorVis7,
+      error: euiTheme.colors.vis.euiColorVis9,
+    };
+  }
+  // Borealis
   return {
-    [RuleTranslationResult.FULL]: euiTheme.colors.vis.euiColorVis0,
-    [RuleTranslationResult.PARTIAL]: euiTheme.colors.vis.euiColorVis5,
-    [RuleTranslationResult.UNTRANSLATABLE]: euiTheme.colors.vis.euiColorVis7,
-    error: euiTheme.colors.vis.euiColorVis9,
+    [RuleTranslationResult.FULL]: euiTheme.colors.vis.euiColorVisSuccess0,
+    [RuleTranslationResult.PARTIAL]: euiTheme.colors.vis.euiColorSeverity7,
+    [RuleTranslationResult.UNTRANSLATABLE]: euiTheme.colors.vis.euiColorSeverity10,
+    error: euiTheme.colors.vis.euiColorSeverity14,
```

Comment:
Just a heads up, the severity colors on Borealis theme might be updated again soon. Please follow the update here: 
https://github.com/elastic/kibana/pull/206276#issuecomment-2592982294

Reply:
Oh good to know. I think that's fine though, we'll update this when colors change